torch/_inductor/bounds.py

```diff
@@ -54,6 +58,7 @@ def get_bounds(self) -> Dict[torch.fx.Node, ValueRanges[Expr]]:
 
         with V.set_ops_handler(ValueRangeAnalysis()):
             interpreter = InterpreterShim(self.loop_body.root_block.graph, submodules)
+            log.debug("get_bounds:\n%s", self.loop_body.root_block.graph)
```

Comment:
I can remove this before actually landing, but this debug was pretty useful working on this PR